_docs/student/account/photo.md

```diff
@@ -5,14 +5,24 @@ redirect_from:
   - /student/account/profile_photo
 ---
 
-You may upload a passport-style photograph of yourself for display to course teaching staff.
+You may upload a passport-style photograph of yourself to display to the course teaching staff.
 This is helpful for learning the names of students in large courses and improving your educational experience.
+Photos uploaded should not exceed 5MB and must be a supported image format (`png`, `jpeg`, `jpg`, `gif`, or `webp`).
 
 ![](/images/student/UploadProfilePhotoForm.png)
 
 Note that this feature is NOT for avatars, memes, etc. Your instructors and the system
 administrators reserve the right to remove profile photos that do not meet educational purposes.
-You will see the following message when your profile photo is found inappropriate and its being flagged by one of 
-your instructors.
 
-![](/images/student/flagged_photo.png)
\ No newline at end of file
+
+If your profile photo is found inappropriate and is flagged by one of your instructors, the following 
+message will be displayed and your profile photo will be replaced or removed.
+
+![](/images/student/flagged_photo.png)
+
+After submitting 50 profile photos, you will not be able to upload any more profile photos and
```

Comment:
I felt like we should say that the limit is 50 photos and rather than saying after submitting 50.

Reply:
Yes sounds good

Reply:
Alright.  I'll update the wording.